services/diddoc_service.go

```diff
@@ -53,6 +53,11 @@ func (dds DIDDocService) Resolve(did string, version string, contentType types.C
 	result := types.DidResolution{Did: &didDoc, Metadata: *resolvedMetadata, ResolutionMetadata: didResolutionMetadata}
 	if didResolutionMetadata.ContentType == types.DIDJSONLD || didResolutionMetadata.ContentType == types.JSONLD {
 		didDoc.AddContext(types.DIDSchemaJSONLD)
+
+		if len(didDoc.Service) > 0 {
+			didDoc.AddContext(types.DIFDIDConfigurationJSONLD)
+		}
+
```

Comment:
Should this additional entry only be returned if the Service type is `LinkedDomains`? A lot of the upstream doc in DID Spec Registries are erroring out. But if I assume that the correct documentation is here [under the `.well-known` extension](https://identity.foundation/.well-known/resources/did-configuration/), then it should _only_ be inserted if and when the service type is `LinkedDomains`.

Tests should also account for and check that this context section is NOT inserted if the service endpoint type is something different.

Reply:
The integration tests cover it already @ankurdotb